hw1/css/style.css

```diff
@@ -0,0 +1,142 @@
+@import 'fonts.css';
+
+
+.weather{ 
+  max-width: 390px;
+  background-color: #f1f8f9;
+  margin: 0 auto;
+  border-radius: 10px;
+  box-shadow: 0px 21px 20.16px 0.84px rgba(0, 0, 0, 0.15);
+}
+
+/* weather top */
+.weather__top {
+  display: flex;
+}
+
+.wt__item {
+  position: relative;
+  flex: 1 1 0;
+  padding: 64px 0 52px;
+  text-align: center;
+  color: #a5acaf;
+}
+.wt__refresh {
+  position: absolute;
+  top: 16px;
+  right: 16px;
+  z-index: 3;  
+  }
+
+/* weather middle */
+.weather__middle {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 18px;
+  color: #a5acaf;
+}
+.wm__item {
+  display: flex;
+  padding: 4px 16px 0 22px;
+  text-align: center;
+}
+.wm__degrees{
+  font-size: 48px;
+  font-family: "Proxima Nova Light";  
+  text-align: center;
+}
+
+.o {
+  font-size: 14px;
+  font-family: "Proxima Nova Regular";
+  color: rgb(141, 147, 150);
+  text-transform: uppercase;
+  position: relative;
+  right: -6px;
+  top: 10px;
+
+}
+.wm__item_middle {
+  text-align: left;
+  margin: 12px 44px 12px 0;
+  font-family: "Source Sans Pro";
+  text-align: left;
+  line-height: 1.2;
+}
+
+.wm__weather{
+  font-size: 24px;
+  font-weight: 300;
+  color: rgb(186, 194, 198);
+}
+
+.wm__location{
+  font-size: 16px;
+  color: rgb(162, 169, 173);
+}
+
+.wm__lastitem{
+  background-color: #54bae6;
+  font-family: "Open Sans Bold";
+  padding: 18px 16px 14px 20px;
+  color: #ffffff;
+}
+.wm__date {
+  font-size: 18px;
+  text-align: center;
+}
+.wm__month {
+  font-size: 14px;
+  line-height: 1.2;
+}
+
+/* weather bottom */
+
+.weather_bottom {
+  background-color: #2e3335;
+  border-bottom-right-radius: 10px; 
+  border-bottom-left-radius: 10px;
+}
+.wb__parameters{padding: 42px 46px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.wb__item {
+  display: flex;
+  text-align: center;
+  color: #525a5d;
+}
+.wb__item:last-child { margin-right: 8px;}
+.wb__text {
+  font-family: "Proxima Nova Bold";
+  font-size: 14px;
+  text-transform: uppercase;
+  padding-left: 20px;
+  color: #ffffff;
+}
+  .wb__scroll{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 18px;
+  }
+.scroll__selected{
```

Comment:
Please fix the indentation and use the same spacing in the whole file. And the same spacing between classes, choose one line or none, it is important to be consisten, to make the code easy to read.
```  
.wb__scroll {
  display: flex;
  justify-content: center;
  align-items: center;
  padding-bottom: 18px;
 }
.scroll__selected {
```